pipeline/pipeline-aws.yml

```diff
@@ -88,25 +87,27 @@ groups:
   jobs:
   - terraform-destroy
 
-resource_types:
 
+resource_types:
 - name: terraform
   type: docker-image
   source:
     repository: ljfranklin/terraform-resource
-    tag: '0.12.6'
+    tag: ((terraform_version))
 
 - name: packer
   type: docker-image
   source:
-    repository: jdub/packer-resource
+    # repository: snapkitchen/concourse-packer-resource
+    repository: sdurrheimer/concourse-packer-resource
```

Comment:
do we need to fork it ? If yes, should it stay on your repo or a cycloid one ?

Reply:
Fixed.
`sdurrheimer/concourse-packer-resource` replaced by `cycloid/concourse-packer-resource`.